airflow/hooks/dbapi_hook.py

```diff
@@ -78,7 +79,7 @@ def get_uri(self) -> str:
         conn = self.get_connection(getattr(self, self.conn_name_attr))
         login = ''
         if conn.login:
-            login = '{conn.login}:{conn.password}@'.format(conn=conn)
+            login = '{quote_plus(conn.login)}:{quote_plus(conn.password)}@'.format(conn=conn)
```

Comment:
This isn't an f-string, so you can't use quote_plus like this.

```suggestion
            login = '{login}:{pass}@'.format(login=quote_plus(conn.login), pass=quote_plus(conn.password))
```

or

```suggestion
            login = f'{quote_plus(conn.login)}:{quote_plus(conn.password)}@'
```